### PR TITLE
Small improvements to eval framework (jsonl, public experiments)

### PR DESCRIPTION
**Pull request by ankrgyl**
* Add support for public experiments through `Eval(...)`
* Add support for `--jsonl` flag to make parsing outputs easy (eg in CI)

### CODE REVIEW

File: js/src/framework.ts [hunk 1]
```diff
@@ -118,12 +136,13 @@ export async function Eval<Input, Output, Expected>(
           progressReporter,
           []
         );
-        reportEvaluatorResult(name, ret, true);
+        reportEvaluatorResult(name, ret, true, false);
```
Comment (manugoyal) [inline js/src/framework.ts:139]: Can we change these to named arguments? Or if backwards-compatibility is a must, add any new arguments as named arguments?

File: py/src/braintrust/framework.py [hunk 2]
```diff
@@ -177,32 +184,42 @@ def pluralize(n, singular, plural):
         return plural
 
 
-def report_evaluator_result(eval_name, results, summary, verbose):
+def report_evaluator_result(eval_name, results, summary, verbose, jsonl):
```
Comment (manugoyal) [inline py/src/braintrust/framework.py:187]: same here, would be nice if we passed by keyword name or made these keyword-only args.

**Review by manugoyal — APPROVED** [review]

File: js/src/framework.ts (hunk 1 above)
Comment (ankrgyl) [reply js/src/framework.ts:139]: fixed

File: py/src/braintrust/framework.py (hunk 2 above)
Comment (ankrgyl) [reply py/src/braintrust/framework.py:187]: fixed

**Pull request merged**